tests/integration_test.py

```diff
@@ -1624,3 +1624,6 @@ def test_649(self):
         ctnr = self.client.create_container('busybox', ['sleep', '2'])
         self.client.start(ctnr)
         self.client.stop(ctnr)
+
+    def test_715(self):
```

Comment:
OK... I'm not sure about that as a general thing. It places a rather unnecessary demand on people reading the code. Out of scope for this PR, but maybe we could rename them with descriptive names and include the URL to the GitHub issue in a docstring?